### PR TITLE
Errors during fetching: FR & EN upgrade proposition

### PR DESCRIPTION
**Pull request by techexo**
I updated the existing content and added a big part on XPath and site config.
When I have your go, I'll translate that in English.

### CODE REVIEW

**Comment by techexo** [comment]
Thanks @j0k3r . Can I begin the translation in English, including your changes?

**Comment by j0k3r** [comment]
Yup go ahead!

**Comment by Kdecherf** [comment]
As a side note, here is a change to use a separate file (`graby.log`) for graby logs in debug mode without changing other channels:

```
monolog:
    handlers:
        main:
            type: fingers_crossed
            action_level: error
            handler: nested
            channels: ['!graby']
        nested:
            type: stream
            path: "%kernel.logs_dir%/%kernel.environment%.log"
            level: debug
        graby:
            type: stream
            path: "%kernel.logs_dir%/graby.log"
            level: debug
            channels: ['graby']
        console:
            type: console
```

**Comment by techexo** [comment]
Thanks @Kdecherf. Do you want me to include that in the doc, or should it stay "confidential"? (I tend to think that the first option is the way to go)

**Comment by Kdecherf** [comment]
Sure, feel free to add it to the documentation. It should be useful ;-)

**Comment by techexo** [comment]
Sorry for all the force-pushing etc., I am not used to work with public repositories and git...
I updated the French documentation to include @Kdecherf change, I think it is a better way to do it that switching everything to debug, even if it needs more change in the `config_prod` file.

File: fr/user/errors_during_fetching.md [hunk 1]
```diff
@@ -45,13 +44,32 @@ Si vous hébergez votre propre instance de wallabag, vous pouvez nous joindre de
...
+            type: console
+```
 - videz le cache avec la commande `rm -rf var/cache/*` ;
-- videz le contenu du fichier de log avec `cat /dev/null > var/logs/prod.log` ;
 - rechargez votre instance wallabag et rechargez le contenu qui pose souci ;
-- si vous ne réussissez pas à déterminer avec les logs l'origine du problème, copiez/collez le contenu du fichier `var/logs/prod.log` dans un nouveau [ticket d'incident GitHub](https://github.com/wallabag/wallabag/issues/new).
+- si vous ne réussissez pas à déterminer avec les logs l'origine du problème, copiez/collez le contenu du fichier `var/logs/graby.log` dans un nouveau [ticket d'incident GitHub](https://github.com/wallabag/wallabag/issues/new).
 
-### Création ou mise à jour d'un fichier de configuration (_site_config_)
+ ### Création ou mise à jour d'un fichier de configuration (_site_config_)
```
[the remark this answers is not in the document]
Comment (techexo) [reply fr/user/errors_during_fetching.md]: Solved in 98c2bb7

File: fr/user/errors_during_fetching.md [hunk 2]
```diff
@@ -183,4 +201,4 @@ Comme de multiples types de paragraphes peuvent contenir un lien `a` ou une bali
 
 Si vous souhaitez plus d'informations sur _XPath_, la norme [contient un résumé assez clair de ce qui est faisable](https://www.w3.org/TR/1999/REC-xpath-19991116/#path-abbrev). Le site **devhints.io** a également [une anti-sèche très complète sur _XPath_](https://devhints.io/xpath).
 
-Si vous voulez tester de manière dynamique si un _XPath_ fonctionne pour sélectionner un ou plusieurs éléments d'une page, vous pouvez également utiliser [ce bac à sable](http://www.whitebeam.org/library/guide/TechNotes/xpathtestbed.rhtm) qui évalue des chemins sur un code HTML/XML que vous pouvez _uploader_.
+ Si vous voulez tester de manière dynamique si un _XPath_ fonctionne pour sélectionner un ou plusieurs éléments d'une page, vous pouvez également utiliser [ce bac à sable](http://www.whitebeam.org/library/guide/TechNotes/xpathtestbed.rhtm) qui évalue des chemins sur un code HTML/XML que vous pouvez _uploader_.
```
[the remark this answers is not in the document]
Comment (techexo) [reply fr/user/errors_during_fetching.md]: Solved in 98c2bb7

**Comment by techexo** [comment]
For information, I am halway there for the English traduction (missing a part of the XPath doc). I will probably push that tonight on the branch, or do you prefer a new branch/PR for that?

**Comment by techexo** [comment]
I pushed the English documentation too if you want to have a look. I did not have time to readproof it yet, but it shouldn't be too bad.

**Comment by techexo** [comment]
@j0k3r @tcitworld @Kdecherf Could someone review this? Or are you planning to merge it with an upcoming release?

**Comment by j0k3r** [comment]
We don't need a new release for that :)

**Pull request merged**